File: ios/Sources/ReleasebirdPlugin/Releasebird.swift
import Foundation
import Releasebird

@objc public class ReleasebirdCapacitor: NSObject {
    @objc public func echo(_ value: String) -> String {
        print(value)
        return value
    }
    
    @objc public func initialize(_ apiKey: String, showButton: Bool) -> Void {
           Releasebird.sharedInstance().initialize(apiKey, showButton: showButton)
    }
    
    @objc public func showWidget() {
        Releasebird.sharedInstance().showWidget();
    }

    @objc public func logout() {
            Releasebird.sharedInstance().logout();
    }
    
    @objc public func identify(_ identifyJson: NSDictionary, andHash: String) -> Void {
        print(identifyJson);
       Releasebird.sharedInstance().identify(identifyJson, hash: andHash);
    }
}

File: android/src/main/java/com/releasebird/capacitor/Releasebird.java
package com.releasebird.capacitor;

import android.app.Application;
import android.util.Log;

import java.util.Map;

public class Releasebird {

    public String echo(String value) {
        Log.i("Echo", value);
        return value;
    }

    public void initialize(String apiKey, Application application) {
        com.releasebird.releasebird_sdk.Releasebird.init(apiKey, application);
    }

    public void showWidget() {
        com.releasebird.releasebird_sdk.Releasebird.getInstance().showWidget();
    }

    public void logout(Application application) {
        com.releasebird.releasebird_sdk.Releasebird.getInstance().logout(application);
    }

    public void identify(Map<String, Object> identifyJson, String hash) {
        com.releasebird.releasebird_sdk.Releasebird.getInstance().identify(identifyJson, hash);
    }



}
